packages/web/docs/src/pages/self-hosting/get-started.mdx

```diff
@@ -89,24 +95,162 @@ the environment variables.
+> Note: deleting this directory will end up with a loss of data, as this directory is acting as a volume for storing the data from Hive dependencies: database and storage
+
+## Testing GraphQL Hive
+
 Visit `http://localhost:8080` in your browser and start using GraphQL Hive! The usage reporting
 endpoint is bound to `http://localhost:8081`. The GraphQL API is bound to `http://localhost:8082`.
 The artifacts bucket is bound to `http://localhost:8083`.
 
+Firstly, you can head to `http://localhost:8080` in your browser and start using GraphQL Hive!
+You'll need to Sign Up for an account by inputting your email and password. And once you do you'll
+see your personal organization that has been automatically created for you by default.
+
+### Creating first project and target
+
+Now that we have GraphQL Hive setup locally, signed up and logged into our dashboard. We can start
+creating our first project which we're gonna push graphql schemas to.
+
+So you're gonna click on the `Create a Project` button on the top right, give it a name and choose
+your project type. You're gonna see all sorts of types `Single` which is the most common and
+there're also `Federation` and `Stitching` but what's also nice is that you can define your own
+custom endpoints for schema validation and schema building. I'm gonna go with `Single` but you can
+choose what best suits your needs. Then click `Create Project`, you're gonna be redirected to your
+project page where you're gonna see 3 listed targets created for you by default:
+
+- `development`
+- `production`
+- `staging`
+
+We're gonna use the `development` target in this guide and as soon as you click it you'll see this
+message:
+
+> Hive is waiting for your first schema.
+>
+> You can publish a schema with Hive CLI and Hive Client
+
+If you already have a Nodejs project with a graphql schema that's great you can skip over the next
+step, but if not so we're gonna initialize a new package in our directory, you can use your
+preferred package manager to do so:
+
+```bash
+# you can do the same with npm or yarn
+pnpm init
+```
+
+Then create a `schema.graphql` file wherever you want, you can put it at the root for example with
+any simple graphql schema like the following:
+
+```graphql
+type Query {
+  hello: String!
+}
+```
+
+You also need to have a git repository initilaized for your project, because Hive automatically uses
+the last commit hash to determine who the author is and ties the schema with the commit hash. So
+we're gonna do that.
+
+```bash
+git init
+```
+
+Make sure you exclude the following in your `.gitignore` file, because you don't wanna push those in
+your commits:
+
+```
+node_modules
+.hive
+hive.json
+.env
+```
+
+Make sure to at least have a single commit so that Hive can associate the schema with it.
+
+```bash
+git add .
+git commit -m"init"
+```
+
+Now that you've your Nodejs project setup, start by installing the GraphQL Hive CLI which we're
+gonna use to interact with Hive's server.
```

Comment:
```suggestion
You also need to have a git repository initilaized for your project, because Hive automatically uses
the last commit hash to determine who the author is and ties the schema with the commit hash. So
we're going to do that.

```bash
git init
```

Make sure you exclude the following in your `.gitignore` file, because you don't wanna push those in
your commits:

```
node_modules
.hive
hive.json
.env
```

Make sure to at least have a single commit so that Hive can associate the schema with it.

```bash
git add .
git commit -m"init"
```

Now that you've your Nodejs project setup, start by installing the GraphQL Hive CLI which we're
going to use to interact with Hive's server.
```